File: MediaPlayerSeven/app/src/main/java/com/jar/mediaplayerseven/MainActivity.java
package com.jar.mediaplayerseven;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

import com.jar.mediaplayerseven.databinding.ActivityMainBinding;
import com.jar.mediaplayerseven.media.FFMediaPlayer;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        // Example of a call to a native method
        TextView tv = binding.sampleText;
        tv.setText(FFMediaPlayer.GetFFmpegVersion());
    }






}

File: MediaPlayerFive/app/src/main/java/com/jar/media_player_five/MainActivity.kt
package com.jar.media_player_five

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import com.jar.media_player_five.databinding.ActivityMainBinding
import com.jar.media_player_five.media.FFMediaPlayer

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        // Example of a call to a native method
        binding.sampleText.text = FFMediaPlayer.GetFFmpegVersion()
    }


    companion object {

    }
}